frontend/src/utils/fieldValidation.ts

```diff
@@ -391,6 +396,16 @@ export const createDateValidationRules: ValidationRuleFn<DateFieldBase> = (
           'Selected date is not within the allowed date range'
         )
       },
+      invaliDays: (val) => {
+        if (!val || !schema.invalidDays || schema.invalidDays.length === 0) {
+          return true
+        }
+
+        return (
+          !isDateAnInvalidDay(parseISO(val), schema.invalidDays) ||
+          'Error placeholder'
```

Comment:
maybe we should think of an error message to put here instead of a placeholder. suggest  
```suggestion
          'Selected date is not an allowed day of the week'
```

Reply:
I was waiting for Pearly to update me with the error message. Let me confirm with her and update it in this PR.

Reply:
Latest commit contains the confirmed error message from Pearly.

Reply:
not in commit yet

Reply:
Broke it down into two commits due to tests I added for util functions

Reply:
3speedy5u